odata-csdl/0 frontmatter.md

```diff
@@ -70,6 +70,10 @@ This specification is related to:
 :
 * _OData JSON Format Version 4.02_. Edited by Ralf Handl, Mike Pizzo, and Heiko Theißen. Latest stage: https://docs.oasis-open.org/odata/odata-json-format/v4.02/odata-json-format-v4.02.html
 
+#### Declared XML namespaces:
+* http://docs.oasis-open.org/odata/ns/edmx
+* http://docs.oasis-open.org/odata/ns/edm
+
```

Comment:
This needs to be conditional:
- CSDL XML needs the XML namespaces
- CSDL JSON needs the JSON Schema

Reply:
But we don't introduce a new namespace in our JSON schema, do we?

Reply:
Correct, I compared with the 4.01 version: only CSDL XML mentions XML namespaces, the JSON Schema is only referenced in Additional Artifacts.